core/pom.xml

```diff
@@ -62,6 +62,14 @@
     </dependency>
     <!-- END: Any23 -->
 
+    <!-- BEGIN: Librdfa-RDF4J, loading from repository -->
+    <dependency>
+        <groupId>${project.groupId}</groupId>
+        <artifactId>apache-any23-librdfa</artifactId>
+        <version>1.0.0</version>
```

Comment:
Ensure that the versioning is consistent with the rest of the codebase e.g. 2.3-SNAPSHOT or whatever it is currently.

Reply:
So, should I include librdfa-rdf4j as part of the build life-cycle?